services/importer.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/leanspace/terraform-provider-leanspace/services/commands/release_queues"
 	"github.com/leanspace/terraform-provider-leanspace/services/dashboard/dashboards"
 	"github.com/leanspace/terraform-provider-leanspace/services/dashboard/widgets"
+	"github.com/leanspace/terraform-provider-leanspace/services/leafSpaceIntegration/contactreservation"
```

Comment:
I would keep snake case convention here so `leaf_space_integration/contact_reservations`
Also I guess it should be plural